integration/tests/basic/opcodes/test_recursion_contract_deploy.py

```diff
@@ -0,0 +1,95 @@
+import allure
+import pytest
+
+from integration.tests.basic.helpers.basic import BaseMixin
+from utils.helpers import generate_text
+
+ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
```

Comment:
we have it in utils https://github.com/neonlabsorg/neon-tests/blob/develop/utils/consts.py#L5